### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by tcitworld**
A *few* changes.

I'll need @wallabag/translators here :p

### CODE REVIEW

**Comment by matteocoder** [comment]
@tcitworld I hope I could work on it this weekend.

File: en/admin/maintenance/restore.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+```
+
+#### PostgreSQL
+```bash
+PGPASSWORD="password" pg_restore -c -d wallabag -h [server] -U [username] wallabag-sqlbkp.bak
+```
+
+#### SQLite
+```bash
+sqlite3 data/owncloud.db < wallabag-sqlbkp.bak
```
Comment (tcitworld) [inline en/admin/maintenance/restore.md]: :/

**Comment by tcitworld** [comment]
Fixed. Translators welcomed.

**Comment by matteocoder** [comment]
@tcitworld Why did you delete the "Backup" section?

**Comment by matteocoder** [comment]
@tcitworld In internal settings there is a phrase: "it allows to handle more sensible settings", I guess that is means more delicate settings, right?

**Pull request closed without merging**